demisto_sdk/commands/doc_reviewer/rn_checker.py

```diff
@@ -63,6 +63,11 @@ class ReleaseNotesChecker:
         'Note: ',
         'Started adoption process.',
         'Completed adoption process.',
+        "Improved layout"
+        "Created a new layout"
+        "Playbook now supports"
+        "Created a new playbook"
+        "Updated the"
```

Comment:
1. Add commas for this to work properly
2. Let's try to keep things uniform and use `'` instead of `"`
```suggestion
        'Improved layout',
        'Created a new layout',
        'Playbook now supports',
        'Created a new playbook',
        'Updated the'
```